backend/fms_core/tests/test_services/sample_test.py

```diff
@@ -135,9 +135,10 @@ def setUp(self) -> None:
 
         # Create samples for testing
         self.SUBMITTED_SAMPLES_TO_POOL = [
-            {"alias": "Alias1", "individual": self.test_individuals[0], "collection_site": "TestSite", "sample_kind": self.sample_kind_dna, "tissue_source": self.sample_kind_blood, "library": self.test_libraries[4], "project": None, "studies": [], "volume": Decimal(10), "experimental_group": None, "fragment_size": 100},
-            {"alias": "Alias2", "individual": self.test_individuals[1], "collection_site": "TestSite", "sample_kind": self.sample_kind_dna, "tissue_source": self.sample_kind_blood, "library": self.test_libraries[5], "project": None, "studies": [], "volume": Decimal(30), "experimental_group": None, "fragment_size": 100},
+            {"alias": "Alias1", "individual": self.test_individuals[0], "collection_site": "TestSite", "sample_kind": self.sample_kind_dna, "tissue_source": self.sample_kind_blood, "library": self.test_libraries[4], "project": None, "studies": [], "volume_ratio": Decimal(0.25), "experimental_group": None, "fragment_size": 100},
+            {"alias": "Alias2", "individual": self.test_individuals[1], "collection_site": "TestSite", "sample_kind": self.sample_kind_dna, "tissue_source": self.sample_kind_blood, "library": self.test_libraries[5], "project": None, "studies": [], "volume_ratio": Decimal(0.75), "experimental_group": None, "fragment_size": 100},
```

Comment:
Might be a good idea to add another service test for sum of ratios not equal to 1.

Reply:
DONE